x/axelarcork/client/proposal_handler.go

```diff
@@ -6,10 +6,10 @@ import (
 )
 
 var (
-	AddProposalHandler                        = govclient.NewProposalHandler(cli.GetCmdSubmitAddCellarIDProposal)
-	RemoveProposalHandler                     = govclient.NewProposalHandler(cli.GetCmdSubmitRemoveCellarIDProposal)
+	AddProposalHandler                        = govclient.NewProposalHandler(cli.GetCmdSubmitAddAxelarCellarIDProposal)
+	RemoveProposalHandler                     = govclient.NewProposalHandler(cli.GetCmdSubmitRemoveAxelarCellarIDProposal)
```

Comment:
Nit: nonspecific variable names

Reply:
Yeah, a lot were changed, but these are internal to the module, it really makes stuff like grepping around a bit more difficult. We principally need stuff that could overlap to be different (e.g. all the proposal commands under `gov` end up in one `submit-proposal` command, so two things with the same CLI string would clash.